### PR TITLE
fixes #19249 - toggle repo sets

### PR DESCRIPTION
**Pull request by thomasmckay**
Allows displaying repo sets three ways:
1. repos available in the subscriptions attached (existing code, unchanged)
2. all repos, regardless of subscriptions (toggle)
3. only repos available in the content view environment (toggle)

### CODE REVIEW

File: app/controllers/katello/api/v2/host_subscriptions_controller.rb [hunk 1]
```diff
@@ -168,7 +168,8 @@ def content_override
     api :GET, "/hosts/:host_id/subscriptions/product_content", N_("Get content and overrides for the host")
     param :host_id, String, :desc => N_("Id of the host"), :required => true
     def product_content
-      content = @host.subscription_facet.candlepin_consumer.available_product_content
+      content_access_mode_all = params[:content_access_mode_all] == 'true'
```
Comment (parthaa) [inline app/controllers/katello/api/v2/host_subscriptions_controller.rb]: ```content_access_mode_all = ::Foreman::Cast.to_bool(params[:content_access_mode_all])```

**Comment by waldenraines** [comment]
UI looks fine to me.

**Comment by thomasmckay** [comment]
@parthaa - rethinking the params, check out this PR for act keys then let's return to this one https://github.com/Katello/katello/pull/6788

**Comment by thomasmckay** [comment]
@parthaa @jlsherrill - updated equivalent repo sets toggles for content hosts as enabled for act keys

**Comment by parthaa** [comment]
Ack . I tested the UI

**Pull request merged**